include/mpm_base.h

```diff
@@ -196,9 +199,10 @@ class MPMBase : public MPM {
   // Level set methods
   bool ls_methods_{false};
   //! Damping type
-  std::string damping_type_{""};
+  enum class Damping{ None, Cundall };
```

Comment:
Define it outside please, otherwise every class has this definition, which is unnecessary